libs/blocks/reading-time/reading-time.js

```diff
@@ -0,0 +1,31 @@
+import { createTag } from '../../utils/utils.js';
+
+function getTextLength(node) {
+  let textLength = 0;
+  for (const childNode of node.childNodes) {
+    if (childNode.nodeType === Node.TEXT_NODE) {
+      if (childNode.textContent.match(/\S+/g)) {
+        const wordCount = childNode.textContent.match(/\S+/g).length;
+        textLength += wordCount;
+      }
+    } else {
+      textLength += getTextLength(childNode);
+    }
+  }
+  return textLength;
+}
+
+
+export default function init(el) {
+  el.innerHTML = '';
+  el.classList.add('content');
+  const span = createTag('span');
+  const readingSpeed = 200; // Assume that the average reader reads 200 words per minute
+  const textLength = getTextLength(document.body);
+  const readingTime = Math.ceil(textLength / readingSpeed);
+  span.innerHTML = `${readingTime} min read`;
```

Comment:
How will this be loc'd?  I think we should have the author define it in the block

Reply:
We can maybe use placeholders so that the author doesn't need to worry about it?

Reply:
Ok I added placeholders in [this commit](https://github.com/adobecom/milo/pull/356/commits/32621253fbbde1df5df89ba70de534a2c9a43f10)

(test URL: https://main--stock--adobecom.hlx.page/es/pages/drafts/solene/loc-spanish-test?milolibs=reading-time)

The placeholder is called `5-min-read` and then the block replaces the number 5 with the correct number

Reply:
Placeholder is a good idea as this will be used on multiple pages.

There should also be at least 2 keys used.  One for singular "1 minute read" and another for 2+ minute plural form.

Reply:
Good catch, it should now work with both singular and plural
Test URL:
https://main--stock--adobecom.hlx.page/es/pages/drafts/solene/loc-spanish-test-2?milolibs=reading-time

Placeholders:
<img width="533" alt="image" src="https://user-images.githubusercontent.com/62023521/211930952-582b8f15-4e05-45d9-a8ac-ffa84af41786.png">